### PR TITLE
Adds .ruby-version file for ruby version auto switch

### PR DESCRIPTION
**Pull request by grzuy**
If you are using rbenv or chruby and your default ruby it's not the one
used by bundler-site, when you clone the project and `bundle`, you get,
e.g. the following error:

`Your Ruby version is 2.3.3, but your Gemfile specified 2.3.1`

We can make this easier for contributors by auto switching ruby version
by defininig `.ruby-version`.

### CODE REVIEW

**Comment by segiddins** [comment]
Please change the gemfile to read from the ruby version file as well, thanks!

**Comment by grzuy** [comment]
Ready :-)

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.3.1'
+ruby File.read('.ruby-version').strip
```
Comment (segiddins) [inline Gemfile]: `File.read(File.expand_path('../.ruby-version', __FILE__))`

**Comment by segiddins** [comment]
This is now on master as of https://github.com/bundler/bundler-site/pull/293/files

**Pull request closed without merging**